www/members/proxy.cgi

```diff
@@ -24,19 +24,18 @@ def emit_instructions(today, cur_mtg_dir, meeting)
   end
   _p %{
     This form allows you to assign a proxy for the upcoming
-    Member's Meeting on #{meeting_display}. If there is any chance you might not be able
-    to attend the first part of the Member's Meeting on Tuesday in IRC, then
-    please assign an attendance proxy, because that helps the meeting reach
-    quorum more quickly - the meeting can't formally continue without quorum at the start.
-    You can still attend the meeting if you want, and you can revoke a
+    Member's Meeting on #{meeting_display}.  Submitting an attendance proxy will
+    help us reach quorum at the start of the meeting - the meeting can't formally
+    continue without quorum at the start.
+
+    You can still vote and attend the meeting if you want, and you can revoke a
```

Comment:
suggested wording:
You will still receive a ballot so you will be able to vote and attend the meeting. You can revoke your